app/services/doc_auth_router.rb

```diff
@@ -144,7 +144,8 @@ def translate_generic_errors!(response)
     end
   end
 
-  def self.client
+  # @param [Proc,nil] warn_notifier proc takes a hash, and should log that hash to events.log
+  def self.client(warn_notifier: nil)
```

Comment:
Just to clarify, since we implement `notify_exception` as a local method in this module; the reason we parameterize it here is since we don't have easy access to analytics, and since `DocumentProofingJob` also needs it for other logging, it's simpler than to `Analytics.new` in both places?

Reply:
yeah that was my thinking, easier to link it to the current user/request than to try to create a from-scratch one